src/lib/components/select/namespace-select.svelte

```diff
@@ -18,7 +18,7 @@
   }
 </script>
 
-<Select bind:value={selectedNamespace} on:change={() => switchNamespace()}>
+<Select dark arrow bind:value={selectedNamespace} on:change={switchNamespace}>
```

Comment:
Do we ever _not_ want that arrow? I wouldn't include it as a parameter to the component because they should all be the same and we'd have to add it everywhere.

Reply:
in the figma the only one with that arrow type is in the top bar